content/docs/support/filing-issues.md

```diff
@@ -0,0 +1,112 @@
+- Bug in a Kubernetes Registry example → [pulumi/pulumi-kubernetes](https://github.com/pulumi/pulumi-kubernetes/issues)
+
+{{% notes type="tip" %}}
+On any Registry page, you can find a link to the provider's repository in the "Repository" section.
+{{% /notes %}}
+
+## Provider and resource issues
+
+If you're experiencing issues with a specific cloud provider or resource (errors, missing features, unexpected behavior), file the issue in that provider's repository.
+
+### Major provider repositories
+
+- **AWS** → [pulumi/pulumi-aws](https://github.com/pulumi/pulumi-aws/issues) or [pulumi/pulumi-aws-native](https://github.com/pulumi/pulumi-aws-native/issues)
+- **Azure** → [pulumi/pulumi-azure-native](https://github.com/pulumi/pulumi-azure-native/issues) or [pulumi/pulumi-azure](https://github.com/pulumi/pulumi-azure/issues)
+- **Google Cloud (GCP)** → [pulumi/pulumi-gcp](https://github.com/pulumi/pulumi-gcp/issues) or [pulumi/pulumi-google-native](https://github.com/pulumi/pulumi-google-native/issues)
+- **Kubernetes** → [pulumi/pulumi-kubernetes](https://github.com/pulumi/pulumi-kubernetes/issues)
+
+### Finding other provider repositories
+
+For other providers, visit the [Pulumi Registry](https://www.pulumi.com/registry/), search for your provider, and click through to its repository.
+
+## Core Pulumi issues
+
+File issues in the main [pulumi/pulumi repository](https://github.com/pulumi/pulumi/issues) for:
+
+- Pulumi CLI bugs or feature requests
+- Pulumi engine issues
+- State management problems (not specific to a provider)
+- Language SDK issues (TypeScript, Python, Go, .NET, Java)
+- Issues with `pulumi new`, `pulumi up`, `pulumi preview`, etc.
+- Pulumi Cloud-related issues (though you can also [contact support](/contact/))
+
+**Examples:**
+
+- The `pulumi up` command crashes
+- State file becomes corrupted
+- Problems with stack references
+- Issues with secrets encryption
+- Language-specific SDK bugs
+
+## Still not sure?
+
+If you're unsure where to file your issue:
+
+1. Start with [pulumi/pulumi](https://github.com/pulumi/pulumi/issues) - the team can redirect you if needed
+1. Join the [Pulumi Community Slack](https://slack.pulumi.com/) and ask in the #general channel
+1. [Contact support](/contact/) if you're on a paid plan
+
+## Before you file
+
+To help us resolve your issue quickly:
+
+1. **Check the troubleshooting guide** - see our [troubleshooting documentation](/docs/support/troubleshooting/) for solutions to common issues
+1. **Search existing issues** - your issue might already be reported
+1. **Include version information** - run `pulumi version` and include the output
+1. **Provide reproduction steps** - minimal code examples are extremely helpful
+1. **Enable verbose logging** - see our [logging guide](/docs/support/debugging/logging/) for instructions
+1. **Redact sensitive information** - remove credentials, tokens, and secrets from logs
```

Comment:
They might also want to redact the org name, depending on whether their company is fussy about revealing what vendors/tools they work with.